quill-core/src/main/scala/io/getquill/Ord.scala

```diff
@@ -0,0 +1,23 @@
+package io.getquill
+
+trait Ord[T]
+
+trait OrdOps {
+
+  def asc[T](implicit ord: Ordering[T]): Ord[T]
+  def desc[T](implicit ord: Ordering[T]): Ord[T]
+  def ascNullsFirst[T](implicit ord: Ordering[T]): Ord[T]
+  def descNullsFirst[T](implicit ord: Ordering[T]): Ord[T]
+  def ascNullsLast[T](implicit ord: Ordering[T]): Ord[T]
+  def descNullsLast[T](implicit ord: Ordering[T]): Ord[T]
+
+  def apply[T1, T2](o1: Ord[T1], o2: Ord[T2]): Ord[(T1, T2)]
+  def apply[T1, T2, T3](o1: Ord[T1], o2: Ord[T2], o3: Ord[T3]): Ord[(T1, T2, T3)]
+  def apply[T1, T2, T3, T4](o1: Ord[T1], o2: Ord[T2], o3: Ord[T3], o4: Ord[T4]): Ord[(T1, T2, T3, T4)]
+  def apply[T1, T2, T3, T4, T5](o1: Ord[T1], o2: Ord[T2], o3: Ord[T3], o4: Ord[T4], o5: Ord[T5]): Ord[(T1, T2, T3, T4, T5)]
+  def apply[T1, T2, T3, T4, T5, T6](o1: Ord[T1], o2: Ord[T2], o3: Ord[T3], o4: Ord[T4], o5: Ord[T5], o6: Ord[T6]): Ord[(T1, T2, T3, T4, T5, T6)]
+  def apply[T1, T2, T3, T4, T5, T6, T7](o1: Ord[T1], o2: Ord[T2], o3: Ord[T3], o4: Ord[T4], o5: Ord[T5], o6: Ord[T6], o7: Ord[T7]): Ord[(T1, T2, T3, T4, T5, T6, T7)]
+  def apply[T1, T2, T3, T4, T5, T6, T7, T8](o1: Ord[T1], o2: Ord[T2], o3: Ord[T3], o4: Ord[T4], o5: Ord[T5], o6: Ord[T6], o7: Ord[T7], o8: Ord[T8]): Ord[(T1, T2, T3, T4, T5, T6, T7, T8)]
+  def apply[T1, T2, T3, T4, T5, T6, T7, T8, T9](o1: Ord[T1], o2: Ord[T2], o3: Ord[T3], o4: Ord[T4], o5: Ord[T5], o6: Ord[T6], o7: Ord[T7], o8: Ord[T8], o9: Ord[T9]): Ord[(T1, T2, T3, T4, T5, T6, T7, T8, T9)]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The number of parameters should not exceed 8](https://www.codacy.com/app/fwbrasil/quill/file/1702455387/issues/source?bid=2746728&fileBranchId=3046083#l21)